changelog_unreleased/javascript/pr-9341.md

```diff
@@ -0,0 +1,31 @@
+#### Improve formatting for `AssignmentExpression` with `ClassExpression` that has a long super class (#9341 by @sosukesuzuki)
+
```

Comment:
You wrote this formatting problem is related to Google Closure library. Would be good to mention that in the changelog too.